docs/how_to/error-mitigation.rst

```diff
@@ -228,7 +219,7 @@ Example of adding ``resilience_options`` into your estimator session
     options.optimization_level = 3
     options.resilience_level = 2
     options.resilience.noise_factors = (1, 2, 3, 4)
-    options.resilience.noise_amplifier = 'CxAmplifier'
+    options.resilience.noise_amplifier = 'LocalFoldingAmplifier'
```

Comment:
We should double check if we want the default value to be `LocalFoldingAmplifier` or just `None`